uber/templates/regform.html

```diff
@@ -15,8 +15,12 @@
     var staffingClicked = function () {
         var checked = $.field('staffing') && $.field('staffing').is(':visible:checked');
         setVisible('#departments', checked);
-        setVisible($.field('no_cellphone').parents('.checkbox'), checked);
-        $('#cellphone').parents('.form-group').find('label:first').css('font-weight', checked ? 'bold' : 'normal');
```

Comment:
Side note: these if-checks are getting increasingly sprinkled throughout the code, which means it might be worth wither changing the semantics of ``$.field('foo')`` to return an empty jQuery collection rather than ``null`` when that field is missing, or to have a separate method that does something similar.

That's a bigger change than I want to make tonight, since at the moment I just want to get rid of these Javascript errors.